tests/default_vars.sh

```diff
@@ -935,6 +935,7 @@ export LNDP_PRT_LIST=-999
 #IAU
 export IAU_INC_FILES="''"
 
+export FH_DFI_RADAR='! (fh_dfi_radar is disabled)'
```

Comment:
See my comment below, we should use the same syntax and logic as for any other entry in the namelist.

Reply:
Testing `export FH_DFI_RADAR='-2e10'` now